<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>700</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Визуализатор L-систем</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="loadButton">
                                <property name="text">
                                    <string>Загрузить L-систему</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="systemNameLabel">
                                <property name="text">
                                    <string>Система не загружена</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QGroupBox" name="controlGroup">
                        <property name="title">
                            <string>Управление эволюцией</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>Текущий шаг:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="stepLabel">
                                            <property name="text">
                                                <string>0</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_3">
                                            <property name="text">
                                                <string>из</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="maxStepLabel">
                                            <property name="text">
                                                <string>0</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QSlider" name="stepSlider">
                                    <property name="minimum">
                                        <number>0</number>
                                    </property>
                                    <property name="maximum">
                                        <number>0</number>
                                    </property>
                                    <property name="value">
                                        <number>0</number>
                                    </property>
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QPushButton" name="prevButton">
                                            <property name="text">
                                                <string>Предыдущий шаг</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="nextButton">
                                            <property name="text">
                                                <string>Следующий шаг</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="resetButton">
                                            <property name="text">
                                                <string>Сброс</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QGroupBox" name="infoGroup">
                        <property name="title">
                            <string>Информация о системе</string>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                            <item>
                                <widget class="QLabel" name="axiomLabel">
                                    <property name="text">
                                        <string>Аксиома: не задана</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="angleLabel">
                                    <property name="text">
                                        <string>Угол: не задан</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="rulesLabel">
                                    <property name="text">
                                        <string>Правила: не заданы</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QWidget" name="drawingArea">
                        <property name="minimumSize">
                            <size>
                                <width>400</width>
                                <height>400</height>
                            </size>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>